internal/entity/query/sessions.go

```diff
@@ -11,7 +11,7 @@ import (
 
 // Session finds an existing session by its id.
 func Session(id string) (result entity.Session, err error) {
-	if l := len(id); l < 6 && l > 2048 {
+	if l := len(id); l < 6 || l > 2048 {
```

Comment:
Good find! I've fixed this in https://github.com/photoprism/photoprism/commit/1e3f9f7984cddc3d8c7e02f440ae7c9b55366eeb.